src/helper/sync.js

```diff
@@ -33,11 +33,8 @@ export default {
     },
     trigger(model, triggerStr, ...args) {
         if (model.trigger) {
-            if (triggerStr.match(/success/)) {
-                model.trigger(triggerStr, args[0]);
-            } else {
-                model.trigger(triggerStr);
-            }
+            args.unshift(triggerStr);
```

Comment:
Instead of this, you could also remove `triggerStr` from the parameters in the function and remove this line. Then the `triggerStr` is part of `args` automatically.